tests/unit/raf-test.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+import { test, moduleForComponent } from 'ember-qunit';
+import wait from 'ember-test-helpers/wait';
+import { generateContent, sortItemsByPosition } from '../helpers/helpers';
+import template from '../templates/fixed-grid';
+
+moduleForComponent('ember-collection', 'raf', {
+  integration: true,
+});
+
+test('works without requestAnimationFrame', function(assert) {
+  var originalRaf = window.requestAnimationFrame;
```

Comment:
I had that but I can't get the teardown to wait for the component to be torn down, so the restoration happens causing the test to fail. Async is hard.

Trying to figure it out now.